test/cppsuite/src/component/statistics/cache_limit.h

```diff
@@ -42,7 +42,9 @@ class cache_limit : public statistics {
     virtual ~cache_limit() = default;
 
     void check(scoped_cursor &cursor) override final;
-    std::string get_value_str(scoped_cursor &cursor) override final;
```

Comment:
get_value_str was a workaround to handle each statistics implementer having their own base type, e.g. double, int64_t, uint64_t. With what we have no we don't need to have this and we still don't cast values.